scripts/pre_commit_linter.py

```diff
@@ -87,6 +87,29 @@
     help='verbose mode. All details will be printed.',
     action='store_true')
 
+EXCLUDED_PHRASES = [
```

Comment:
Why this change?

Reply:
Because I needed these lists in the `MANDATORY_PATTERNS_REGEXP` and `MANDATORY_PATTERNS_JS_REGEXP`, so they had to be moved up. I also did this so that all the pattern lists are grouped together. Let me kno wif you think this was a mistake.
Thanks

Reply:
Makes sense. Thanks for the explanation.